src/server/game/Handlers/PetHandler.cpp

```diff
@@ -59,64 +60,64 @@ void WorldSession::HandleDismissCritter(WorldPacket& recvData)
     }
 }
 
-void WorldSession::HandlePetAction(WorldPacket& recvData)
+void WorldSession::HandlePetAction(WorldPackets::Pet::PetAction& packet)
 {
-    ObjectGuid guid1;
-    uint32 data;
-    ObjectGuid guid2;
-    float x, y, z;
-    recvData >> guid1;                                     //pet guid
-    recvData >> data;
-    recvData >> guid2;                                     //tag guid
-    // Position
-    recvData >> x;
-    recvData >> y;
-    recvData >> z;
-
-    uint32 spellid = UNIT_ACTION_BUTTON_ACTION(data);
-    uint8 flag = UNIT_ACTION_BUTTON_TYPE(data);             //delete = 0x07 CastSpell = C1
+   // ObjectGuid guid1;
+   // uint32 data;
+   // ObjectGuid guid2;
+   // float x, y, z;
+   // recvData >> guid1;                                     //pet guid
+   // recvData >> data;
+   // recvData >> guid2;                                     //tag guid
+   // Position
+   // recvData >> x;
+   // recvData >> y;
+   // recvData >> z;
+
+   // uint32 spellid = UNIT_ACTION_BUTTON_ACTION(data);
+   // uint8 flag = UNIT_ACTION_BUTTON_TYPE(data);             //delete = 0x07 CastSpell = C1
```

Comment:
If this code is no longer needed remove it entirely instead of commenting it out.